addons/hr_timesheet/models/project.py

```diff
@@ -147,6 +147,20 @@ def write(self, values):
                 raise UserError(_('This task must be part of a project because there are some timesheets linked to it.'))
         return super(Task, self).write(values)
 
+    @api.multi
+    def name_get(self):
+        names = super().name_get()
```

Comment:
Not needed here, We can move to end for simpler return

Reply:
Not going to work. Goal is to append data to the end of the name string. That variable is needed like 2 lines below.

Reply:
I see new changes here and seems better than before. :+1: